examples/BERT/data.py

```diff
@@ -0,0 +1,603 @@
+import torch
```

Comment:
I guess we could start incrementally porting this over into the experimental folder which will also help with cleanup. These datasets seem useful in general.

Reply:
I second this. I would also help make this pull request shorter.